### DISCUSSION TITLE
How can I release storage space?

### DISCUSSION THREAD

**Discussion by hwzolin** [home-assistant/operating-system#2657] [Q&A]
Hello, 
Now I hane a problem, my hoome assistant storage shows that disk metrics used space 94.2%. The space is almost filled up.
![full space](https://github.com/home-assistant/operating-system/assets/42085859/74256db6-fbfc-4a0f-ae8b-4930463c938a)
I don't know how to clean up unused data to free up enough space.
Please help me,
Thank you.

**Comment by JohnMcLear** [comment]
TLDR; it's a bug.  See:

https://github.com/home-assistant/supervisor/issues/4148
https://github.com/home-assistant/supervisor/issues/3223
